### ISSUE TITLE
Problem while trying to install latest versions.

### ISSUE THREAD

**Issue by sh-dv** [covert-encryption/covert#53]
Cool project first off all.

Running `pip install covert` downloads version 0.2.1 which is weird.
I thought maybe i have a cached old version of covert so i tried `--no-cache-dir --force-reinstall` commands. Same result, v 0.2.1)

Running `pip install covert==0.5.3` shows the following error : 
```
ERROR: Could not find a version that satisfies the requirement covert==0.5.3 (from versions: 0.2.0, 0.2.1)
ERROR: No matching distribution found for covert==0.5.3
```

This issue maybe is only on my computer but i thought maybe it's something worth checking.

In the homepage i prefer if you change the pip install command to `pip install "covert[gui]"`  (with double Quotation marks).

In addition, writing a good Installation guide for covert must be done, and a guide on how to build/run the code from source, Dev environment, etc. This will save a lot of time for interested people to take a look at the project instead of spending some time and effort to figure how to run it. Good luck!

Thanks!

**Comment by covert-encryption** [comment]
Good points on documentation, especially for developers. This product is not currently intended for end users, so that part might come later. I added the double quotes (required on bash, I guess).

The version problem is very odd. Sometimes I had to use `--upgrade` to get the latest but at least that is downloading 0.5.3 on various systems that I just tried. Will have a look at PyPI later to find out if anything is different with the 0.2.x versions.

**Comment by covert-encryption** [comment]
@sh-dv Could you confirm which Python version you are running? The requirement for python>=3.9 was added to setup.py in 0.3.0, and this is probably causing pip not to find those packages (I would appreciate a clearer message on that).

Python 3.9 is still quite new and some distributions or stock MacOS might be shipping older releases but you can probably find 3.9 and 3.10 as separate packages, and can then use `python3.9 -m pip install covert`.

**Comment by sh-dv** [comment]
I am actually running 3.10, which apparently was updated yesterday, usually python is up to date on my system.
However, i debugged the pip install and it showed that those packages were pulled from the 3.8 python version directory when i did the pip install command.
I have removed previous python versions (3.8+3.9) and their site-packages directory. did a restart and the pip install command went fine and installed latest version of cover.
